projects/golang/go/build/conformanceTests/builderBaseRebuildBuildspec.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+env:
+  variables:
+    ARTIFACTS_BUCKET: "eks-d-postsubmit-artifacts"
+    BUILDKIT_PLATFORMS: "linux/amd64"
+    GO_SOURCE_VERSION: "1.16"
```

Comment:
this is going to change -- right now, I just have one pipeline which builds based on these buildspecs. However, before this is all wrapped up, we'll do this differently -- either by having a pipeline PER GO VERSION or having a pipeline with a build project per Go version. This env var will then be set in those pipeliens/proejcts.